fmriprep/config.py

```diff
@@ -110,6 +113,16 @@
 
 DEFAULT_MEMORY_MIN_GB = 0.01
 
+# Ping NiPype eTelemetry once if env var was not set
+# workers on the pool will have the env variable set from the master process
+if _nipype_et is None:
```

Comment:
I don't think we should ping here if the user has explicitly set either:
1) `NIPYPE_NO_ET`
or
2) `NO_ET` (generic)

Reply:
That's the trick - if the user has set `NIPYPE_NO_ET` this won't ping. I am not checking `NO_ET`, but that'd be easy to do.